### ISSUE TITLE
RSS links get served as text/html

### ISSUE THREAD

**Issue by mrdomino** [dwrensha/wordpress-sandstorm#26]
On my private site, if I visit /feed/, I get a response content-type of application/rss+xml. On the public site, that link gets rewritten as /feed/index.html and served as text/html. This seems weird. I can't tell if this is bad or not, aside from causing Chrome to try to render it.

**Comment by ocdtrekkie** [comment]
For what it's worth, I think most web browsers will try to provide a navigable rendered UI for XML files these days. But probably only if they can tell it's XML.

**Comment by mrdomino** [comment]
There's definitely a difference in Chrome between the two content types: visiting /feed on my private site shows XML source, whereas /feed on the public site shows a badly mangled rendering of the XML as though it were HTML.

**Comment by mrdomino** [comment]
(Re-reading your comment, I now think you were saying that it's okay to serve up the content as XML. I'd agree.)

**Comment by dwrensha** [comment]
I think the issue here is that [our wget hack for generating static content](https://github.com/dwrensha/wordpress-sandstorm/blob/master/publish-it.sh) is not smart enough to properly handle XML.

**Comment by mrdomino** [comment]
I'm now tempted to rewrite that in Python.

**Comment by dwrensha** [comment]
It's possible that wget has learned some new tricks in the past two years and now has a nice way to deal with this kind of thing.

**Comment by paulproteus** [comment]
IMHO we should use a WordPress plugin that is specifically designed for turning a WordPress site static, such as:

- https://wordpress.org/plugins/simply-static/

- https://wordpress.org/plugins/really-static/

rather than rolling our own solution that will run into WordPress-specific subtleties that we will have to reinvent solutions for.

**Comment by mrdomino** [comment]
Assuming that's the road we want to go down, I'm certainly willing and possibly able to make time to do it. @dwrensha what do you think?

**Comment by dwrensha** [comment]
Yeah, it'd be great if we could get an existing plugin to do the hard work for us. Last time I tried to go that route, I think I ran into difficulties regarding directory structure, and maybe I also hit sqlite-vs-mysql problems. But it seems like there ought to be a way to make it work, so I encourage you to try.

**Comment by paulproteus** [comment]
FWIW, I also personally think that MySQL would be a fine database for the Sandstorm WordPress package, to maximize compatibility. That's a separate topic to discuss on a separate issue perhaps.

**Comment by xet7** [comment]
This issue was moved to wpsandstorm/wordpress-sandstorm#20
